File: prop-all/src/test/java/io/pleo/prop/objects/InvalidJSON.java
package io.pleo.prop.objects;

import javax.inject.Inject;
import javax.inject.Named;

import io.pleo.prop.core.Prop;

public class InvalidJSON {
  @Inject
  public InvalidJSON(@Named("io.pleo.prop5") Prop<InjectedObject> object) {

  }
}

File: prop-all/src/test/kotlin/io/pleo/prop/objects/InvalidJSON.kt
package io.pleo.prop.objects

import com.google.inject.Inject
import com.google.inject.name.Named
import io.pleo.prop.core.Prop

@Suppress("UNUSED_PARAMETER")
class InvalidJSON
@Inject
constructor(
    @Named("io.pleo.prop5") objectProp: Prop<InjectedObject>,
)
